Changelog.md

```diff
@@ -12,6 +12,7 @@ Bugfixes:
  * isoltest: Added new keyword `wei` to express function value in semantic tests
  * Standard-JSON-Interface: Fix a bug related to empty filenames and imports.
  * SMTChecker: Fix internal errors when analysing tuples.
+ * solc: Throwing stack too depp ICE in case of calling encode with too many arguments
```

Comment:
Actually, since it's only an "improved ICE", I would skip the changelog entry for this one :-). Otherwise this looks good!

Reply:
Removed, updating